network-api/networkapi/wagtailpages/factory/publication.py

```diff
@@ -10,14 +14,34 @@
 
 
 class PublicationPageFactory(PageFactory):
-    title = Faker('text', max_nb_chars=255)
+    title = Faker('text', max_nb_chars=120)
 
     class Meta:
         model = PublicationPage
 
 
+class ArticlePageFactory(PageFactory):
+    title = Faker('text', max_nb_chars=120)
+
+    class Meta:
+        model = ArticlePage
+
+
 def generate(seed):
     reseed(seed)
     home_page = get_homepage()
+    """
+    reate a couple scenarios that will be best for testing: 
+    * A PublicationPage with several child ArticlePages
+    * A PublicationPage with child PublicationPages, each of which has their own ArticlePages
+        * perhaps nested at random levels of depth?
+    """
 
     PublicationPageFactory.create_batch(parent=home_page, size=3)
+    # single PublicationPage with several child Articles
+    ArticlePageFactory.create_batch(parent=PublicationPage.objects.all()[0], size=8)
```

Comment:
Could we throw `PublicationPage.objects.all()` into a variable and reference that instead of `PublicationPage.objects.all()[0]` and `PublicationPage.objects.all()[1]` each time?

Reply:
definitely! i mainly left it that way because there's probably a better way to do it overall and wanted the tech debt to obvious (at least to myself)